packages/atomic/cypress/integration/facets/color-facet/color-facet-selectors.ts

```diff
@@ -11,6 +11,12 @@ export const ColorFacetSelectors = {
     ColorFacetSelectors.shadow().find(
       '[part="value-box"][aria-pressed="false"]'
     ),
+  selectedBoxValueWithText: (text: string) =>
+    ColorFacetSelectors.shadow().find(
+      `[part="value-box"][aria-pressed="true"]:contains("${text}")`
+    ),
+  idleBoxValueLabel: () =>
+    ColorFacetSelectors.idleBoxValue().find('[part="value-label"]'),
```

Comment:
These two selectors (with equivalents for `checkbox` and `link`) were added to every facet. They are used by `selectIdle*ValueAt` tests.